packages/tags/README.md

```diff
@@ -0,0 +1,31 @@
+# @zendeskgarden/react-tags [![npm version](https://img.shields.io/npm/v/@zendeskgarden/react-tags.svg?style=flat-square)](https://www.npmjs.com/package/@zendeskgarden/react-tags) [![Dependency Status](https://img.shields.io/david/zendeskgarden/react-components.svg?path=packages/tags&style=flat-square)](https://david-dm.org/zendeskgarden/react-components?path=packages/tags) <!-- markdownlint-disable -->
+
+<!-- markdownlint-enable -->
+
+This package includes components relating to tags in the
+[Garden Design System](https://zendeskgarden.github.io/).
+
+This package includes components relating to tags in the
```

Comment:
Remove duplicate.

Reply:
It's so good it needed to be said twice. Fixed.